_posts/2022-03-15-kubeflow-1.5-release.md

```diff
@@ -0,0 +1,52 @@
+
+## Simplified operations
+
+v1.5 simplifies operations in several ways including easier support of high availability for the Katib Controller via the new hyperparameter leader [election](https://github.com/kubeflow/katib/pull/1713).  v1.5 also adds the MPI framework to the Unified Training Operator, which  enables users to leverage a single distributed training CRD for the most popular frameworks: Tensorflow, Pytorch, MXNet, XGBoost and MPI.  Kubeflow Pipelines (KFP) has incorporated support for the [Emissary Executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) as the default executor, which means that users are not required to manually configure this option to Emissary.  This enhancement also enables KFP to support the newer Kubernetes versions that require a Container Runtime Interface (CRI) and provides support for Docker containers.  
+
+In addition, KServe v0.7 is integrated with Kubeflow v1.5.  For users who are still using KFServing v0.6.1, v1.5 does support the installation of the KFServing v0.6.1 release.  For users migrating from KFServing to KServe, there is helpful documentation, which you can find [here](https://kserve.github.io/website/0.7/admin/migration/).  From a feature standpoint, KServe v0.7 has added a beta of [ModelMesh](https://github.com/kserve/modelmesh-serving), which enables easier scaling of model serving and provides an architecture that overcomes pod and IP limitations that limit the number of models that can run on a single node and/or cluster.
+
+Note on K8s 1.22: Although many of the Kubeflow services (KFP, AutoML, Training Operator, KServe) have started testing with K8 1.22, v1.5 was not thoroughly tested with K8s 1.22.   Additionally, critical work is still outstanding for Kubeflow Notebooks and other central services.  While the Community always encourages testing, K8s 1.22 is not formally supported with Kubeflow v1.5.  K8s 1.22 support is on the roadmap for Kubeflow’s next release and if you are interested in K8s 1.22 support, please add a comment on this tracking [issue](https://github.com/kubeflow/kubeflow/issues/6098).
+
+## Streamlined user experience
+
+In v1.5, the Kubeflow User Interface (UI) dashboards for Notebooks, AutoML and Volume Manager more closely match KFP’s dashboard.   These enhancements include how dashboard fields are displayed and how operations are performed.  This gives users a consistent look and feel when working in the UI.   Additionally, Kubeflow notebooks and the volume manager have implemented a new configuration template for configuring and using persistent volumes and Kubernetes storage classes.  This template makes it easier to expose more storage classes and will enable easier support of popular storage offerings, including NFS.  In addition, AutoML has also updated its SDK, CI framework and parameter settings across frameworks (goptuna, optuna, hyperopt).
```

Comment:
I support the suggestion above.    I am ok with not defining PVC or NFS.   If we want the last sentence to use parallel sentence structure, then it might read...This will give end users the ability to modify crucial parts of the PVCs, such as StorageClasses, and to more easily support popular storage offerings, including NFS.

Reply:
I addressed both of your comments.